source/PyMaterialX/PyMaterialXCore/PyElement.cpp

```diff
@@ -14,8 +14,6 @@
 
 #define BIND_ELEMENT_FUNC_INSTANCE(T)                                                                           \
 .def("_addChild" #T, &mx::Element::addChild<mx::T>)                                                             \
-.def("_getChildOfType" #T, &mx::Element::getChildOfType<mx::T>)                                                 \
-.def("_getChildrenOfType" #T, &mx::Element::getChildrenOfType<mx::T>, py::arg("category") = mx::EMPTY_STRING)   \
```

Comment:
@bernardkwok @jstone-lucasfilm I need some guidance on these binding api 
`template <class T> shared_ptr<T> getChildOfType(ConstDocumentPtr datalibrary, const string& name) const`
`template <class T> shared_ptr<T> getChildOfType(const string& name) const`

Reply:
@ashwinbhat Initially I think it should be fine to omit these new Python bindings, while we're working out the details of the new API, and we can add these once the C++ API is locked down.